ytmusicapi/__init__.py

```diff
@@ -2,14 +2,16 @@
 
 from ytmusicapi.setup import setup, setup_oauth
 from ytmusicapi.ytmusic import YTMusic
+from .mixins.browsing import Lyrics, TimedLyrics, LyricLine
```

Comment:
I was thinking similar, but then you'll need to create a submodule for these classes to be accessible

Reply:
You can already access them: ``from ytmusicapi.mixins.browsing import TimedLyrics``

It's similar for https://github.com/sigma67/ytmusicapi/blob/main/ytmusicapi/parsers/podcasts.py

Although with #621 in mind, it might be wise to move them to a `ytmusicapi.models` module (one for `podcasts` and one for `lyrics`, I guess)